src/main/java/com/github/onsdigital/babbage/configuration/Configuration.java

```diff
@@ -138,6 +143,29 @@ public static Integer getElasticSearchPort() {
         public static String getElasticSearchCluster() {
             return elasticSearchCluster;
         }
+
+        /**
+         * Method to load the list of retired product pages to be hidden
+         * @return List of url strings containing the black listed urls
+         */
+        public static List<String> getHighlightBlacklist() {
+            ClassLoader classLoader = Configuration.class.getClassLoader();
+            File file = new File(classLoader.getResource(HIGHLIGHT_URL_BLACKLIST_FILE).getFile());
+
+            List<String> urls = new ArrayList<>();
+            try (BufferedReader bw = new BufferedReader(new FileReader(file))) {
+                String url;
+
+                while ((url = bw.readLine()) != null) {
+                    urls.add(url);
+                }
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
I know the logging is crap in Babbage but can you add a sout message giving the context i.e. "error while attempting to load highlight blacklist file" - will make it easier to see from the "logs" what the issue is.